gsuite_reports_rules/gsuite_drive_external_share.yml

```diff
@@ -0,0 +1,270 @@
+AnalysisType: rule
+Filename: gsuite_drive_external_share.py
+RuleID: GSuite.Drive.ExternalFileShare
+DisplayName: External GSuite File Share
+Enabled: true
+LogTypes:
+  - GSuite.Reports
+Tags:
+  - GSuite
+  - Security Control
```

Comment:
Should this be disabled and tagged `Configuration Required` since the user needs to define their company domain (this bit `COMPANY_DOMAIN = 'your-company-name.com'`).

Reply:
Yep

Reply:
Looks like this is still enabled